changelog/bug-1859323.md

```diff
@@ -0,0 +1,5 @@
+audience: worker-deployers
+level: minor
```

Comment:
Yeah, I was torn between saying it was a bug and a feature, but since it doesn't misbehave or do something wrong/bad, it just doesn't tell you what the working dir is, I figured it was a feature. I don't mind either way.